package.json

```diff
@@ -18,6 +18,7 @@
   "author": "the UI5 community <do_not_use@openui5.org> (https://github.com/ui5-community)",
   "type": "module",
   "exports": {
+    "types": "./esm/index.d.ts",
```

Comment:
is the identical entry on line 25 `"types": "esm/index.d.ts"` still needed or can it be safely deleted?

Reply:
See my comment above:

> Note that this is different from the types field at the top-level of package.json, which is a legacy mechanism for the old module resolution.

`moduleResolution` `Node16` makes use of the new node mechanisms, which means respecting the `exports` field as documented at https://nodejs.org/api/packages.html#exports. The CommonJS-build explicitly uses the legacy moduleResolution, which does not look at `exports` -- here I'm not sure, but I guess this is what the legacy top-level `types` was used for.

See above where I link to another project that switched completely and tries to explain the reasoning. See also the wdio-built-in project that also dual-packages and keeps both fields (also linked above)